app/Models/Activity.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Activity extends Model
```

Comment:
This Pull Request contains changes already included in the `main` branch.

Maybe pull changes into this branch?